example/nuxt.config.js

```diff
@@ -1,7 +1,6 @@
-import _module from '..'
-
 export default {
   modules: [
-    _module
-  ]
+    '../src/module.ts'
+  ],
+  watch: ['../templates/*']
```

Comment:
Doesn't it cause full nuxt restart on template change?

Reply:
```suggestion
```